lua/prettier/options.lua

```diff
@@ -43,7 +43,9 @@ local default_options = {
     "yaml",
   },
   ["null-ls.condition"] = function()
-    return u.config_exists()
+    return u.config_exists({
+      check_package_json = true,
```

Comment:
Let's change the readme to say the opposite of what it says now, i.e.:

```
        -- if `false`, skips checking `package.json` for `"prettier"` key
        check_package_json = true,
```

The intention is to document the default value and mention what happens if user changes it.

Reply:
Makes sense, have updated.